runtime/src/main/jni/JsV8InspectorClient.cpp

```diff
@@ -298,3 +296,4 @@ jclass JsV8InspectorClient::inspectorClass = nullptr;
 jmethodID JsV8InspectorClient::sendMethod = nullptr;
 jmethodID JsV8InspectorClient::sendToDevToolsConsoleMethod = nullptr;
 jmethodID JsV8InspectorClient::getInspectorMessageMethod = nullptr;
+int JsV8InspectorClient::contextGroupId = 1;
```

Comment:
please leave comment on why it's left "1" and when will that need to possibly change.